### ISSUE TITLE
version 1.0.7+ fails to launch on system with Intel 3rd Gen Core processor Graphics

### ISSUE THREAD

**Issue by opbillbobaggins** [pop-os/cosmic-settings#1935]
Launching cosmic-settings from terminal output:
`cosmic-settings
  ERROR  Failed to read config 'accessibility_zoom', err: GetKey("accessibility_zoom", Os { code: 2, kind: NotFound, message: "No such file or directory" })
    at cosmic-settings/src/pages/accessibility/magnifier.rs:60 on main

  ERROR  shortcuts custom config error: GetKey("custom", Os { code: 2, kind: NotFound, message: "No such file or directory" })
    at /build/cosmic-settings-icFwkj/cosmic-settings-1.0.7~1774389977~24.04~7dc9530/vendor/cosmic-settings-config/src/shortcuts/mod.rs:43 on main

  ERROR  Failed to load Panel system config., err: GetKey("padding_overlap", Os { code: 2, kind: NotFound, message: "No such file or directory" })
    at cosmic-settings/src/pages/desktop/panel/mod.rs:90 on main

  ERROR  Error when loading Panel container config., err: GetKey("padding_overlap", Os { code: 2, kind: NotFound, message: "No such file or directory" })
    at cosmic-settings/src/pages/desktop/panel/inner.rs:71 on main

  ERROR  Error when loading Panel container config., err: GetKey("padding_overlap", Os { code: 2, kind: NotFound, message: "No such file or directory" })
    at cosmic-settings/src/pages/desktop/panel/inner.rs:71 on main

  ERROR  Failed to load Dock system config., err: GetKey("padding_overlap", Os { code: 2, kind: NotFound, message: "No such file or directory" })
    at cosmic-settings/src/pages/desktop/dock/mod.rs:138 on main

  ERROR  Error when loading Panel container config., err: GetKey("padding_overlap", Os { code: 2, kind: NotFound, message: "No such file or directory" })
    at cosmic-settings/src/pages/desktop/panel/inner.rs:71 on main

  ERROR  Error when loading Panel container config., err: GetKey("padding_overlap", Os { code: 2, kind: NotFound, message: "No such file or directory" })
    at cosmic-settings/src/pages/desktop/panel/inner.rs:71 on main

  ERROR  Failed to read config 'focus_follows_cursor', err: GetKey("focus_follows_cursor", Os { code: 2, kind: NotFound, message: "No such file or directory" })
    at cosmic-settings/src/pages/desktop/window_management.rs:53 on main

  ERROR  Failed to read config 'cursor_follows_focus', err: GetKey("cursor_follows_focus", Os { code: 2, kind: NotFound, message: "No such file or directory" })
    at cosmic-settings/src/pages/desktop/window_management.rs:61 on main

  ERROR  Failed to read config 'focus_follows_cursor_delay', err: GetKey("focus_follows_cursor_delay", Os { code: 2, kind: NotFound, message: "No such file or directory" })
    at cosmic-settings/src/pages/desktop/window_management.rs:70 on main

  ERROR  Failed to read config 'active_hint', err: GetKey("active_hint", Os { code: 2, kind: NotFound, message: "No such file or directory" })
    at cosmic-settings/src/pages/desktop/window_management.rs:79 on main

  ERROR  Failed to read config 'edge_snap_threshold', err: GetKey("edge_snap_threshold", Os { code: 2, kind: NotFound, message: "No such file or directory" })
    at cosmic-settings/src/pages/desktop/window_management.rs:88 on main

  ERROR  shortcuts custom config error: GetKey("custom", Os { code: 2, kind: NotFound, message: "No such file or directory" })
    at /build/cosmic-settings-icFwkj/cosmic-settings-1.0.7~1774389977~24.04~7dc9530/vendor/cosmic-settings-config/src/shortcuts/mod.rs:43 on main

  ERROR  Failed to read config 'workspaces', err: GetKey("workspaces", Os { code: 2, kind: NotFound, message: "No such file or directory" })
    at cosmic-settings/src/pages/desktop/workspaces.rs:44 on main

  ERROR  Failed to read config 'input_default', why: GetKey("input_default", Os { code: 2, kind: NotFound, message: "No such file or directory" })
    at cosmic-settings/src/pages/input/mod.rs:53 on main

  ERROR  Failed to read config 'workspaces', err: GetKey("workspaces", Os { code: 2, kind: NotFound, message: "No such file or directory" })
    at cosmic-settings/src/pages/input/touchpad.rs:27 on main

  ERROR  Failed to read config 'descale_xwayland', err: GetKey("descale_xwayland", Os { code: 2, kind: NotFound, message: "No such file or directory" })
    at cosmic-settings/src/pages/applications/legacy_applications.rs:61 on main

  ERROR  Failed to read config 'xwayland_eavesdropping', err: GetKey("xwayland_eavesdropping", Os { code: 2, kind: NotFound, message: "No such file or directory" })
    at cosmic-settings/src/pages/applications/legacy_applications.rs:70 on main

MESA-INTEL: warning: Ivy Bridge Vulkan support is incomplete
MESA-INTEL: warning: Ivy Bridge Vulkan support is incomplete
  ERROR  Handling wgpu errors as fatal by default
    at /build/cosmic-settings-icFwkj/cosmic-settings-1.0.7~1774389977~24.04~7dc9530/vendor/wgpu/src/backend/wgpu_core.rs:1058 on main

The application panicked (crashed).
Message:  wgpu error: Validation Error

Caused by:
  In Device::create_shader_module, label = 'iced_wgpu.quad.solid.shader'

Shader validation error: Function [1] 'unpack_u32' is invalid
   ┌─ iced_wgpu.quad.solid.shader:9:1
   │
 9 │ ╭ fn unpack_u32(data: vec2<u32>) -> vec4<f32> {
10 │ │     let rg: vec2<f32> = unpack2x16float(data.x);
   │ │                         ^^^^^^^^^^^^^^^ naga::ir::Expression [2]
11 │ │     let ba: vec2<f32> = unpack2x16float(data.y);
12 │ │
13 │ │     return vec4<f32>(rg.y, rg.x, ba.y, ba.x);
   │ ╰─────────────────────────────────────────────^ naga::ir::Function [1]
   │
   = Expression [2] is invalid
   = Shader requires capability Capabilities(SHADER_FLOAT16_IN_FLOAT32)

      Expression [2] is invalid
        Shader requires capability Capabilities(SHADER_FLOAT16_IN_FLOAT32)

Location: /build/cosmic-settings-icFwkj/cosmic-settings-1.0.7~1774389977~24.04~7dc9530/vendor/wgpu/src/backend/wgpu_core.rs:1058

Backtrace omitted. Run with RUST_BACKTRACE=1 environment variable to display it.
Run with RUST_BACKTRACE=full to include source snippets.
Aborted (core dumped)`

**Comment by LiamillionSS** [comment]
Does this issue still happen on the latest 1.0.9?

**Comment by opbillbobaggins** [comment]
Not yet available in apt update
